lib/preprocessed/ToString.post.sol

```diff
@@ -56,6 +56,10 @@ function toString(LocalUnpacked memory __unpacked) pure returns (string memory)
   return string.concat("Local{","active: ", vm.toString(__unpacked.active), ", ", "fee: ", vm.toString(__unpacked.fee), ", ", "density: ", toString(__unpacked.density), ", ", "tickPosInLeaf: ", vm.toString(__unpacked.tickPosInLeaf), ", ", "level0: ", toString(__unpacked.level0), ", ", "level1: ", toString(__unpacked.level1), ", ", "level2: ", toString(__unpacked.level2), ", ", "kilo_offer_gasbase: ", vm.toString(__unpacked.kilo_offer_gasbase), ", ", "lock: ", vm.toString(__unpacked.lock), ", ", "last: ", vm.toString(__unpacked.last),"}");
 }
 
+function tickBranchToString(Tick tick) pure returns (string memory) {
+  return string.concat(vm.toString(tick.posInLevel2()), "->", vm.toString(tick.posInLevel1()), "->", vm.toString(tick.posInLevel0()), "->", vm.toString(tick.posInLeaf()));
```

Comment:
I would go even further and give the indices of each level!

Reply:
Addressed in 69f8b815afb701420ae5537f9639656fca2a5bf7